packages/fluentui/react-northstar/src/themes/teams-dark/components/Label/labelVariables.ts

```diff
@@ -6,11 +6,41 @@ export const labelVariables = (siteVars: SiteVariablesPrepared): Partial<LabelVa
   return {
     colorScheme: pickValuesFromColorScheme(
       extendColorScheme(siteVars.colorScheme, {
+        default: {
+          background: 'rgb(255, 255, 255)',
+          foreground: 'rgba(45, 44, 44, 1)',
+        },
+        black: {
+          foreground: siteVars.colorScheme.black.foreground1,
+          background: siteVars.colorScheme.black.background,
+        },
+        white: {
+          foreground: siteVars.colorScheme.white.foreground,
+          background: siteVars.colorScheme.white.background1,
+        },
         brand: {
-          background: siteVars.colorScheme.brand.foreground4,
+          foreground: siteVars.colorScheme.brand.foreground5,
+          background: siteVars.colorScheme.brand.background5,
+        },
+        grey: {
+          foreground: siteVars.colorScheme.grey.foreground7,
+          background: siteVars.colorScheme.grey.background6,
+        },
+        orange: {
+          foreground: siteVars.colorScheme.orange.foreground2,
+          background: siteVars.colorScheme.orange.background,
         },
         red: {
-          background: siteVars.colorScheme.red.foreground1,
+          foreground: siteVars.colorScheme.red.foreground1,
+          background: siteVars.colorScheme.red.background,
+        },
+        green: {
+          foreground: siteVars.colorScheme.green.foreground3,
+          background: siteVars.colorScheme.green.background,
+        },
+        yellow: {
+          foreground: siteVars.colorScheme.yellow.foreground3,
+          background: siteVars.colorScheme.yellow.background,
```

Comment:
Can we please remove `colorScheme` magic there and perform a small refactor?

`labelVariables.ts`

```tsx
yellowForeground: siteVars.colorScheme.yellow.foreground3,
yellowBackground: siteVars.colorScheme.yellow.background,
```

`labelStyles.ts`

```
...p.color === 'yellow' && {
  color: v.yellowForeground,
  backgroundColor: v.yellowBackground,
}
```

Please contact me offline if something is unclear